kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/FileSystemMetadataProvider.java

```diff
@@ -295,8 +295,8 @@ public boolean exists(String namespace, String name) {
 
   @SuppressWarnings("deprecation")
   @Override
-  public List<String> namespaces() {
-    List<String> namespaces = Lists.newArrayList();
+  public Set<String> namespaces() {
```

Comment:
So no one has to look this up, I confirmed that the interfaces require `Collection` so `Set` or `List` is fine.